packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx

```diff
@@ -256,4 +259,12 @@ export class StackedBarWidget {
       this.el.forceUpdate();
     }
   }
+
+  // If container is not ready or the widget is invisible block drawing phase
+  private _containerIsNotDrawable() {
+    if (!this.container || this.el.clientWidth === 0 || this.el.clientHeight === 0) {
```

Comment:
Do we have `this.el` always available when asking for their properties?

Reply:
We should, it's a decorated class property that gets assigned the DOM node of the element.